packages/@rnw-scripts/integrate-rn/src/test/logger.test.ts

```diff
@@ -59,17 +59,17 @@ test('Console Output', () => {
 
   logger.close();
 
-  expect(stripAnsiColors(loggerOuptput)).toBe(
-    `i Commits: foo
+  expect(stripAnsiColors(loggerOuptput)).toMatch(
```

Comment:
`ora` will specialize output of some symbols dependent on OS (because Windows traditionally supports less chars). I was running this on a Mac locally, so I just replaced those special characters with wildcards so that we can run fuller xplat UT coverage.